ros/lib/rbac_interface.py

```diff
@@ -158,18 +159,26 @@ def set_host_groups(rbac_response):
 
     role_list = rbac_response['data']
     host_groups = []
```

Comment:
If you want to avoid having duplicate values in `host_groups`, you may want to use a Set instead of a List

Reply:
done!